docs/refarch/boundedContexts/security/index.md

```diff
@@ -1,21 +1,71 @@
-# {name} BC
+# Security BC
+
+1. Overview
+2. User / Operator login (AuthN)
+3. Authorization Model (AuthZ)
+4. BC bootstrap 
+5. BC startup
+6. Role / Privilege association
+7. Example Query / call
 
-{overview}
 
 ## Terms
 
-Terms with specific and commonly accepted meaning within the Bounded Context in which they are used.
+Terms with specific and commonly accepted meaning within the Security Bounded Context.
 
-| Term | Description |
+| Crypto providers | Adapter that will provide cryptographic services and Key Management System |
 |---|---|
-| Term1 | Description1 |
+| IAM | Identity and Access Management. Adapter that will provide services for menus, profiles, roles and permissions.  |
```

Comment:
I would suggest adding the following terms to this table:

- AuthN
- AuthZ
- JWT
- KMS